versioned_docs/version-4.5/developers/components/built-in.md

```diff
@@ -20,19 +19,6 @@ Harper provides extended features using built-in components. They do **not** nee
 
 <!-- ## clustering -->
 
-## fastifyRoutes
```

Comment:
We likely want to keep this part as we'll still support fastify through this built in and there are customers actively using it on this version too. We aren't removing fastify support yet; just don't want to promote its use anymore